linkerd/app/inbound/src/policy/store.rs

```diff
@@ -120,14 +120,18 @@ where
             self.cache
                 .get_or_insert_with(dst.port(), |port| match self.discover.clone() {
                     Some(disco) => info_span!("watch", port).in_scope(|| {
+                        tracing::trace!(%port, "spawning policy discovery");
```

Comment:
if `trace` is enabled, the port _should_ already be present on the span, which is `info`, right?